### PR TITLE
webgpu: fix NaN warning

### PR DESCRIPTION
**Pull request by xhcao**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5927)

### CODE REVIEW

**Comment by xhcao** [comment]
@qjia7 @axinging @haoyunfeix @gyagp Please take a look, thank you.

File: tfjs-backend-webgpu/src/kernels/binary_op_util.ts [hunk 1]
```diff
@@ -143,7 +143,7 @@ const POW_VEC4 = `
   if (isExpZero.a) {
     resultTemp.a = 1.0;
   }
-  let isNaN = vec4<f32>(a < vec4<f32>(0.0)) * vec4<f32>(floor(b) < b);
+  let isNaN = a < vec4<f32>(0.0) & floor(b) < b;
```
Comment (qjia7) [inline tfjs-backend-webgpu/src/kernels/binary_op_util.ts:146]: Why the check of `isNaN` is different with L161?

**Comment by gyagp** [comment]
Can we change the name to isnan instead of isNanCustom?

File: tfjs-backend-webgpu/src/kernels/binary_op_util.ts (hunk 1 above)
Comment (xhcao) [reply tfjs-backend-webgpu/src/kernels/binary_op_util.ts:146]: `a` and `b` are the different parameters for `pow` and `minumin/maxmium`, I think these two operators have different behaviors for NaN.

**Review by xhcao — COMMENTED** [review]
@gyagp Do you mean `isNan` or `isnan`? If `isNan`, it is a built-in function, it will be a compiling error when running tfjs on old chrome, or adding this built-in function back.
I think `isNanCustom` is good name.

**Comment by xhcao** [comment]
@gyagp It seems that WGSL do NOT support macro declaration, so cannot define an macro as GLSL.  Is it OK that use the name `isNanCustom` currently?

**Comment by gyagp** [comment]
> @gyagp It seems that WGSL do NOT support macro declaration, so cannot define an macro as GLSL. Is it OK that use the name `isNanCustom` currently?

ok for now.

**Review by gyagp — APPROVED** [review]
LGTM

**Comment by gyagp** [comment]
@lina128 @pyu10055 FYI

**Pull request merged**